### ISSUE TITLE
Dragging issue

### ISSUE THREAD

**Issue by habib3962** [amcharts/amcharts4#3093]
**Question**
Why on Dragging right after zooming a little, dot position goes to extreme right?

**Environment (if applicable)**

**Additional context**
https://codepen.io/habib3962/pen/BazMNRV?editors=0010
i have created a pin and if default dateAxis zoom is set to almost middle.
but when you zoom and try to goes to right with dragging, dot remain on right side of chart.
i want to get its position back to center when i dragged chart to most right.

**Comment by zeroin** [comment]
We will fix this problem in next release. As you zoom in the axis to more than 1, to make it work properly you will have to specify:
```
chart.cursor.maxPanOut = 4;
dateAxis.maxZoomDeclination = 4;
```

**Comment by martynasma** [comment]
Fixed in [4.10.12](https://github.com/amcharts/amcharts4/releases/tag/4.10.12).

## [4.10.12] - 2020-11-26

### Changed
- `DateAxis` setting `timezone` is being deprecated. Please consider setting `timezone` on chart or axis `dateFormatter`. `DateAxis` will now properly place grid in time zones that use non-whole hour fractions, e.g. ("Australia/Adelaide").
- Accessibility: Roles and tabindexes in `ExportMenu` elements has been altered to better comply with ARIA standards.

### Fixed
- Scrollbar could act funky when repeatedly clicking on its grips.
- A minor conflict with Google Charts lib on IE fixed.
- Improved performance of `ValueAxis` scale calculation.
- Setting `min` and `max` on a `DateAxis` could result in excessive overzoom and an empty grid in some cases.
- Overzooming `DateAxis` coul dresult in some extra space shown on it.
- Legend was dynamically adding items on a Treemap chart when drilling-down.
- When Legend's labels' `width` were set in percent, they were not being properly sized.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
